apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -157,16 +170,46 @@ export const siteRouter = router({
       })
 
       await db.transaction().execute(async (tx) => {
-        await tx
+        const user = await tx
+          .selectFrom("User")
+          .where("id", "=", ctx.user.id)
+          .selectAll()
+          .executeTakeFirstOrThrow()
```

Comment:
[nit] can we throw a custom error? otherwise this might reflect as 5xx

Reply:
Updated in 489502187e9c9f571dc75a6b344c5ea98ce31990